documentation.html

```diff
@@ -61,6 +61,70 @@ <h2>Components</h2>
         <!-- main content gose here -->
         <main>
             <section class="middle-section-bar">
+                <div id="badge">
+                    <div class="components-intro">
+                        <h1>Badge</h1>
+                        <p>Badges are being used to display a notification count or status information.</p>
+                    </div>
+                    <div class="components-desc">
+                        <div class="components-desc-badge">
+                            <div class="badge-container">
+                                <div class="avatar big-size-avatar flex-items">
+                                    <img class="round-img big-avatar" src="avatar.jpg" alt="avtar">
+                                </div>
+                                <span class="badge-status status-online"></span>
+                            </div>
+                            <div class="badge-container">
+                                <div class="avatar md-size-avatar flex-items">
+                                    <img class="round-img md-avatar" src="avatar.jpg" alt="avtar">
+                                </div>
+                                <span class="badge-status status-offline"></span>
+                            </div>
+                            <div class="badge-container">
+                                <div class="avatar sm-size-avatar flex-items">
```

Comment:
you are following badge-container , like this convention.. then next you are naming your classes as md-size-avatar .. could be named as badge-container convention.

Reply:
ok i will resolve that